### PR TITLE
Update creating-a-github-pages-site-with-jekyll.md

### PR DESCRIPTION
**Pull request by oschihin**
### Why:

Completes information that has been misleading in my setup.

### What's being changed:

Short extension to git commands

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@oschihin 
Thanks for opening a PR :+1: Please fill out "why" and "what's being changed" in the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, so I can get this up for review!

**Comment by ramyaparimi** [comment]
@oschihin 
Thanks so much for filling out the PR template! I got this triaged for review :zap:

File: content/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll.md [hunk 1]
```diff
@@ -68,7 +68,9 @@ Before you can use Jekyll to create a {% data variables.product.prodname_pages %
  If you chose to publish your site from the `gh-pages` branch, create and checkout the `gh-pages` branch.
  ```shell
  $ git checkout --orphan gh-pages
- # Creates a new branch, with no history or contents, called gh-pages and switches to the gh-pages branch
+ # Creates a new branch with no history, called gh-pages and switches to the gh-pages branch
+ # If your repository had content and you want to remove it from the gh-pages branch, do
+ $ git rm -rf .
```
Comment (lecoursen) [inline content/pages/setting-up-a-github-pages-site-with-jekyll/creating-a-github-pages-site-with-jekyll.md:73]: `git checkout --orphan` does create a branch with no contents. You might be seeing the files in your working directory (not yet committed to your branch) and staging area.

```suggestion
 # Creates a new branch, with no history or contents, called gh-pages, and switches to the gh-pages branch
 $ git rm -rf 
 # Removes the contents from your default branch from the working directory
```

**Review by lecoursen — COMMENTED** [review]
Thanks @oschihin! I am going to apply a suggestion to correct a slight technical inaccuracy and then merge. ⚡

**Review by lecoursen — APPROVED** [review]

**Pull request merged**